examples/perplexity/perplexity.cpp

```diff
@@ -458,23 +458,24 @@ static bool decode_helper(llama_context * ctx, llama_batch & batch, std::vector<
     return true;
 }
 
+#define K_TOKEN_CHUNK 4
```

Comment:
add a comment to describe what it does and then merge

Reply:
Don't think it's necessary